openmdao/components/indep_var_comp.py

```diff
@@ -23,6 +23,9 @@ class IndepVarComp(Component):
     def __init__(self, name, val=None, **kwargs):
         super(IndepVarComp, self).__init__()
 
+        if 'promotes' in kwargs:
+            raise ValueError('IndepVarComp init: promotes is not supported in IndepVarComp.')
+
```

Comment:
ah.. you're right.. I was looking at the text of the story...  IndepVarComp is a special case because it has the kwargs.. disregard my comment.